doppelganger/bayesnets.py

```diff
@@ -335,8 +336,8 @@ def define_bayes_net_structure(nodes, edges):
     node_to_index = {name: i for i, name in enumerate(nodes)}
     structure = []
     for child in nodes:
-        structure.append(tuple(node_to_index[parent] for parent in child_to_parents[child]))
-    return tuple(tuple(s) for s in structure)
+        structure.append(frozenset(node_to_index[parent] for parent in child_to_parents[child]))
+    return tuple(frozenset(s) for s in structure)
```

Comment:
`test_load_bayes_net` was failing in Python 3 due to the third tuple sorting as (1, 0) instead of (0, 1). I didn't see any suggestions that bayes net parents needed to be ordered, so I thought maybe using frozensets instead of tuples would solve the problem and make it clear that order isn't necessary - but you can definitely change it back!